dev-tools/action-install-fluvio-cluster.sh

```diff
@@ -1,27 +1,15 @@
 #!/bin/bash
```

Comment:
We should move action related files into it's own directory.  maybe `actions`?

Reply:
Hmm. `actions` works. I was also thinking about doing `.gh-actions/` but I don't love that either.